mopidy_tunein/tunein.py

```diff
@@ -360,6 +361,11 @@ def search(self, query):
     @cache()
     def _tunein(self, variant, args):
         uri = (self._base_uri % variant) + '?render=json' + args
+        #TODO: if config filters are set add it
+        if (self._filter == 'stations'):
```

Comment:
We've also ready got these [strings](https://github.com/kingosticks/mopidy-tunein/blob/develop/mopidy_tunein/translator.py#L13-L14) and the [mapping the other way](https://github.com/kingosticks/mopidy-tunein/blob/develop/mopidy_tunein/translator.py#L82-L91).

Reply:
Yes of course 👍